en-US/Docs/PinMappingsDb.md

```diff
@@ -172,14 +172,19 @@ using Windows.Devices.I2c;
 
 public async void I2C()
 {
-	var settings = new I2cConnectionSettings(0x40); /* 0x40 is the I2C device address   */
-	settings.BusSpeed = I2cBusSpeed.FastMode;       /* FastMode = 400KHz                */
-
-	I2cDevice Device = await I2cDevice.GetDefault(settings);    /* Create an I2cDevice with the default bus controller (I2C0) and the supplied I2C settings */
-
-	byte[] WriteBuf = new byte[] { 0x01, 0x02, 0x03, 0x04}; /* Some data to write to the device */
-
-	Device.Write(WriteBuf);
+    // 0x40 is the I2C device address
+    var settings = new I2cConnectionSettings(0x40);
+    // FastMode = 400KHz
+    settings.BusSpeed = I2cBusSpeed.FastMode;
+
+    // Create an I2cDevice with the specified I2C settings
```

Comment:
Dragonboard has two I2C controllers, so it's undefined which one GetDefault() will return. Instead use the GetDeviceSelector/FindAllAsync/FromIdAsync patter:
string aqs = I2cDevice.GetDeviceSelector("I2C0");
var dis = await DeviceInformation.FindAllAsync(aqs);
using (I2cDevice device = I2cDevice.FromIdAsync(dis[0].Id)) {
...